pkgs/misc/vim-plugins/vim-plugin-names

```diff
@@ -394,6 +394,7 @@ nvim-treesitter/nvim-treesitter-textobjects
 nvim-treesitter/playground
 ocaml/vim-ocaml
 octol/vim-cpp-enhanced-highlight
+ojroques/vim-oscyank@main
```

Comment:
is the `@main` needed?

Reply:
yes, without that `update.py` fails with 404 atom feed request.